examples/tanstack-start/src/routes/deferred.tsx

```diff
@@ -4,10 +4,10 @@ import { Suspense, useState } from "react"
 import { personServerFn, slowServerFn } from "~/functions/deferred"
 
 export const Route = createFileRoute("/deferred")({
-  loader: async () => {
+  loader: async ({ context }) => {
     return {
       deferredStuff: new Promise<string>((r) =>
-        setTimeout(() => r("Hello deferred!"), 2000)
+        setTimeout(() => r(context.i18n.t("Hello deferred!")), 2000)
```

Comment:
Please use `msg` macro, search for other places i changed. 

The reason why it should be `msg` and not the string directly into `t()` function is because `_` (alias `t`) is accepting an ID not a message. Where you use `msg` macro it creates id from your message. 

```js
// examples of correct usage of both
context.i18n.t("deferred.page.greeting")
context.i18n.t(msg`Hello deferred!`)
```
Generally i would not recommend mixing and matching "id based" approach with "natural language based approach"